src/main/java/com/wynntils/modules/core/instances/inventory/FakeInventory.java

```diff
@@ -108,7 +108,8 @@ private void close(InventoryResult result) {
         FrameworkManager.getEventBus().unregister(this);
         isOpen = false;
 
-        if (windowId != -1) McIf.mc().getConnection().sendPacket(new CPacketCloseWindow(windowId));
+        // No need to send close window packet on overlap, window was already closed.
+        if (windowId != -1 && result != InventoryResult.CLOSED_OVERLAP) McIf.mc().getConnection().sendPacket(new CPacketCloseWindow(windowId));
```

Comment:
Condition is now quite big and has a comment so better to add brackets!
```suggestion
        if (windowId != -1 && result != InventoryResult.CLOSED_OVERLAP) {
            McIf.mc().getConnection().sendPacket(new CPacketCloseWindow(windowId));
        }
        
```